CriteoPublisherSdk/Sources/Criteo.m

```diff
@@ -105,28 +101,19 @@ - (void)registerCriteoPublisherId:(NSString *)criteoPublisherId
 - (void)_registerCriteoPublisherId:(NSString *)criteoPublisherId
                        withAdUnits:(NSArray<CRAdUnit *> *)adUnits {
   self.config.criteoPublisherId = criteoPublisherId;
-
-  CR_CacheAdUnitArray *cacheAdUnits = [CR_AdUnitHelper cacheAdUnitsForAdUnits:adUnits];
-  [self.registeredAdUnits addObjectsFromArray:cacheAdUnits];
   [self.appEvents registerForIosEvents];
   [self.appEvents sendLaunchEvent];
   [self.configManager refreshConfig:self.config];
   if (!self.config.liveBiddingEnabled) {
```

Comment:
Not part of this PR, but shouldn't we remove the prefetch only after an A/B test ?

Reply:
This does not remove the prefetch, just disable it when live bidding is enabled? Which is in fact part of the A/B testing, isn't?

Reply:
As discussed, will submit another PR to remove that check.